tools/cuttlefish-host-image-installer/metapackage-google/usr/share/cf_docker_co_run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Step 1. Build Cuttlefish docker image if there is no such one
+cf_image=$(docker image list | grep "cuttlefish-orchestration")
+if [ "$cf_image" == "" ]; then
+  # Build CF docker image
+  cd /home/vsoc-01
+  if [ ! -d "/home/vsoc-01/android-cuttlefish" ]; then
```

Comment:
It seems strange that if the path `/home/vsoc-01/android-cuttlefish` exists, then getting into somewhere in `/home/vsoc-01/android-cuttlefish-stable`. Have you checked both conditions?

Reply:
Thanks! My bug.
It used to be "/home/vsoc-01/android-cuttlefish" when I use git command. But git command keeps getting issues (can't download codes completely), therefore, I changed to use wget command. But the wget command has different folder name "/home/vsoc-01/android-cuttlefish-stable".

I am fixing it. Thanks!

Reply:
I think it looks good to me after fixing remaining comment. I'll change my review status, in order not to be blocked by my approval.